### ISSUE TITLE
dist/cdn version is broken by recent changes to lodash/lodash

### ISSUE THREAD

**Issue by jung-thomas** [juanjoDiaz/json2csv#57]
## Description
We are using json2csv in the browser via the dist/cdn.  The library, BaseParser.js, in turn loads get.js from lodash: https://github.com/juanjoDiaz/json2csv/blob/3f075bd158665e4121964e10e03fd2366e0b5e45/dist/cdn/plainjs/BaseParser.js#L2
Recent commit to the lodash/lodash project removes the get.js file you are using: https://github.com/lodash/lodash/commit/97d4a2fe193a66f5f96d7b813bed96b35b5abf15
This results in json2csv being unusable via the dist/cdn in the browser unfortunately.

## Filing an issue

1. Include the version of json2csv used and the interface that you are using (plainjs, node or whatwg).
7.0.3 in the browser.  Loading via:
```JavaScript
              import Parser  from 'https://cdn.jsdelivr.net/gh/juanjoDiaz/json2csv@7.0.3/dist/cdn/plainjs/Parser.js'
              import flatten  from 'https://cdn.jsdelivr.net/gh/juanjoDiaz/json2csv@7.0.3/dist/cdn/transforms/flatten.js'
```

2. Include your node version or your browser vendor and version.
Browser: Microsoft Edge 117.0.2045.31 (Official build) (64-bit)

3. Include the command or code you used.
We received the error just trying to import the library. Dev tools shows the 404 on the load of the dependent lodash from BaseParser.js
![image](https://github.com/juanjoDiaz/json2csv/assets/12159356/43e18a51-5a32-434a-bc86-a5e5f194cfd4)

4. Include a sample dataset that we can test against.
We don't get that far due to the error loading lodash

**Comment by jung-thomas** [comment]
See PR #58 - for a possible work around to this issue.

**Comment by juanjoDiaz** [comment]
Fixed since v7.0.4
